packages/app-mobile/components/screens/ConfigScreen.tsx

```diff
@@ -341,7 +379,7 @@ class ConfigScreenComponent extends BaseScreenComponent {
 		);
 	}
 
-	renderButton(key: string, title: string, clickHandler: Function, options: any = null) {
+	renderButton(key: string, title: string, clickHandler: ClickListener, options: any = null) {
```

Comment:
Switching to a `()=> void` type was made necessary by switching from `const { ... } = require('react-native')` to `import { ... } from 'react-native'`.

Reply:
Can we get the actual type from react-native? I'd prefer we don't define types for external libraries